src/hardware/sblaster.cpp

```diff
@@ -2388,121 +2441,139 @@ static uint8_t ctmixer_read()
 	// if ( sb.mixer.index< 0x80) LOG_MSG("Read mixer %x",sb.mixer.index);
 
 	switch (sb.mixer.index) {
-	case 0x00: // RESET
+	case 0x00: // ReseT
```

Comment:
```suggestion
	case 0x00: // Reset
```
Or was this deliberate?

Reply:
Nope, it's a mistake 😄